packages/select/src/FilterMultiSelect/components/Trigger/TriggerButtonBase/TriggerButtonBase.tsx

```diff
@@ -10,16 +10,19 @@ import styles from "./TriggerButtonBase.module.scss"
 export type TriggerButtonBaseProps = {
   children: React.ReactNode
   classNameOverride?: string // TODO: migrate it to use OverrideClassName<T> and omit the props controlled by React-Aria
+  isDisabled?: boolean
```

Comment:
Hey @kaydwithers  could you change the prop to `disabled`?  Our rational is that this would be closer to extending native button props

Reply:
Thanks @mcwinter07, that's done now.

Reply:
Thanks @kaydwithers  just confirming you caught other comment on the PR?

Reply:
@mcwinter07 yes, TriggerButtonBase does not have a story. We are using it as a subcomponent and custom styling it on our end.